styles/trinket/catppuccin.user.css

```diff
@@ -0,0 +1,1280 @@
+      }
+
+      & .trinket-wrapper .trinket-content {
+        background-color: @base !important;
+      }
+
+      &,
+      & .trinket-content-wrapper > :not(.codeOutput, #content-overlay) {
+        background-color: @base;
+      }
+
+      & .codeOutput {
+        background-color: initial !important;
+      }
+
+      & .left-menu-toggle {
+        border-right-color: @surface0 !important;
+      }
+
+      & .right-menu-toggle {
+        border-left-color: @surface0 !important;
+      }
+
+      & .icon-bar .item.split {
+        &.button,
+        &.faux-button {
+          & span {
+            border-left-color: @surface0 !important;
+          }
+
+          & span::after {
+            border-top-color: @text !important;
+          }
+        }
+      }
+    }
+
+    #content-overlay {
+      background-color: rgba(
+        red(@base),
+        green(@base),
+        blue(@base),
+        0.2
+      ) !important;
+    }
+
+    #books-index .book-img {
+      outline-color: @surface0 !important;
+
+      &:hover {
+        outline-color: @accent-color !important;
+      }
+    }
+
+    #svg-logo .letters {
+      fill: @text !important;
+    }
+
+    #svg-logo .yellow {
+      fill: @yellow !important;
+    }
+
+    #svg-logo .blue {
+      fill: @blue !important;
+    }
+
+    #svg-logo .green {
+      fill: @green !important;
+    }
+
+    #svg-logo .white {
+      fill: @base !important;
+    }
+
+    #blue-logo-circle {
+      // There normally isn't an animation applied to this
+      // element because the circle blends in to the background.
+      opacity: 0;
+      animation: RtL 1s 0.75s ease 1 forwards !important;
+    }
+
+    #hero {
+      background-color: @mantle !important;
+    }
+
+    #dragbar {
+      background-color: @surface0 !important;
+      border-left-color: @surface0 !important;
+      border-right-color: @surface0 !important;
+    }
+
+    #output-dragbar {
+      background-color: @surface0 !important;
+      border-top-color: @surface0 !important;
+      border-bottom-color: @surface0 !important;
+    }
+
+    #graphic {
+      // This is the element that is the code output for Python graphics (and probably more)
+      // If styled with catppuccin, it may make previous code projects unviewable because of
+      // color contrast. This can still be overwritten (at least in python) by using
+      // `turtle.Screen().bgcolor("some color")` from the `turtle` module.
+      background-color: white !important;
+
+      &,
+      & label {
+        color: initial !important;
+      }
+    }
+
+    // Animation names override default just for color change
+    @keyframes attention-error-border-pulse {
+      0% {
+        box-shadow: inset 0 0 0 @red;
+      }
+      40% {
+        box-shadow: inset 0 0 0 @red;
+      }
+      70% {
+        box-shadow: inset 0 0 6px @red;
+      }
+      100% {
+        box-shadow: inset 0 0 0 @red;
+      }
+    }
+
+    @keyframes bluePulseButton {
```

Comment:
@uncenter can we turn this linter warning off.

Reply:
Yes I already have it changed locally but I forgot to PR it yesterday :)